From 0ef20bdc57622e57d85996ae7ecfbbe3dff5c3a6 Mon Sep 17 00:00:00 2001
From: Patrick Combes <patrick.combes@intel.com>
Date: Tue, 20 Aug 2013 09:16:49 +0200
Subject: Revert "[PORT FROM MAIN][BT] Firmware file different on ffrd and on vv"

BZ: 117478

This reverts commit 87773f2f1db17f55fafd4278f5a1dcbebe8f460c.

Conflicts:

	conf/intel/baylake/bt_vendor.conf
	conf/intel/bodegabay/bt_vendor.conf
	conf/intel/ctp7160/bt_vendor.conf
	conf/intel/ctpscalelt/bt_vendor.conf
	conf/intel/saltbay_pr0/bt_vendor.conf
	conf/intel/saltbay_pr1/bt_vendor.conf
	conf/intel/victoriabay/bt_vendor.conf

Change-Id: I3c16072332ac6cfb321839f15b5c271d7c5b259b
Signed-off-by: Patrick Combes <patrick.combes@intel.com>
Reviewed-on: http://android.intel.com:8080/126532
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 src/conf.c     |    2 --
 src/hardware.c |   30 ------------------------------
 2 files changed, 0 insertions(+), 32 deletions(-)

diff --git a/src/conf.c b/src/conf.c
index 2592e57..2a88093 100644
--- a/src/conf.c
+++ b/src/conf.c
@@ -37,7 +37,6 @@
 ******************************************************************************/
 int userial_set_port(char *p_conf_name, char *p_conf_value, int param);
 int hw_set_patch_file_path(char *p_conf_name, char *p_conf_value, int param);
-int hw_set_patch_file_root_path(char *p_conf_name, char *p_conf_value, int param); // Added by intel to be able to handle board revision
 int hw_set_patch_file_name(char *p_conf_name, char *p_conf_value, int param);
 #if (VENDOR_LIB_RUNTIME_TUNING_ENABLED == TRUE)
 int hw_set_patch_settlement_delay(char *p_conf_name, char *p_conf_value, int param);
@@ -75,7 +74,6 @@ typedef struct {
 static const conf_entry_t conf_table[] = {
     {"UartPort", userial_set_port, 0},
     {"FwPatchFilePath", hw_set_patch_file_path, 0},
-    {"FwPatchFileRootPath", hw_set_patch_file_root_path, 0}, // Added by intel to be able to handle board revision
     {"FwPatchFileName", hw_set_patch_file_name, 0},
 #if (VENDOR_LIB_RUNTIME_TUNING_ENABLED == TRUE)
     {"FwPatchSettlementDelay", hw_set_patch_settlement_delay, 0},
diff --git a/src/hardware.c b/src/hardware.c
index d57ac0b..212789c 100644
--- a/src/hardware.c
+++ b/src/hardware.c
@@ -1443,36 +1443,6 @@ int hw_set_patch_file_path(char *p_conf_name, char *p_conf_value, int param)
 
 /*******************************************************************************
 **
-** Function        hw_set_patch_file_root_path
-**
-** Description     Set the root location of firmware patch files. Then, the
-**                 board revision (e.g. "vv", "pr") etc is appended to this.
-**                 (created by intel)
-**
-** Returns         0
-**
-*******************************************************************************/
-int hw_set_patch_file_root_path(char *p_conf_name, char *p_conf_value, int param)
-{
-    // Buffer to get the value of the property giving the board revision
-    char board_revision[PROPERTY_VALUE_MAX] = {0};
-
-
-    /* First, copy the root file path received as parameter */
-    strcpy(fw_patchfile_path, p_conf_value); // as in hw_set_patch_file_path()
-
-    /* Then get and  append the board revision (e.g. vv, pr etc) */
-    property_get("ro.spid.wifi.nvram", // TODO: rename the property giving the board revision
-                 board_revision,
-                 "vv" /* if no property is set, assume it is a vv board */);
-    strcat(fw_patchfile_path, board_revision);
-    /* no need to append a final slash: hw_config_findpatch() copes with this. */
-
-    return 0;
-}
-
-/*******************************************************************************
-**
 ** Function        hw_set_patch_file_name
 **
 ** Description     Give the specific firmware patch filename
-- 
1.7.4.1

